example/example.py

```diff
@@ -45,7 +46,8 @@ def post(self):
 
     def callback(self, res, error=None):
         if error is not None:
-            self.finish({"error": error})
+            self.finish({"error": str(error)})
+            return
```

Comment:
revert all "empty" changes to initial statement. If I understand correctly, only this line need to resolve an issue.